### PR TITLE
Deprecating Mobile Tutorials in favor of ExecuTorch

### PR DESCRIPTION
**Pull request by AlannaBurke**
## Description
Redirects the following to Executorch:

Model Preparation for Android Recipe
(Prototype) Convert Mobilenetv2 to Core ML
Image Segmentation DeepLabV3 on Android
Image Segmentation DeepLabV3 on iOS
Summary of PyTorch Mobile Recipes
Model Preparation for iOS Recipe
(beta) Efficient mobile interpreter in Android and iOS
(Prototype) Use iOS GPU in PyTorch
Pytorch Mobile Performance Recipes
(Beta) Convert MobileNetV2 to NNAP

## Checklist

- [ ] The issue that is being fixed is referred in the description (see above "Fixes #ISSUE_NUMBER")
- [ ] Only one issue is addressed in this pull request
- [ ] Labels from the issue that this PR is fixing are added to this pull request
- [ ] No unnecessary issues are included into this pull request.

### CODE REVIEW

File: beginner_source/deeplabv3_on_android.rst [hunk 1]
```diff
@@ -1,234 +1,10 @@
 Image Segmentation DeepLabV3 on Android
 =================================================
 
-**Author**: `Jeff Tang <https://github.com/jeffxtang>`_
+This tutorial has been deprecated. There is a new tutorial on this topic.
```
Comment (svekars) [inline beginner_source/deeplabv3_on_android.rst:4]: We should probably say something like: PyTorch Mobile is no longer actively supported. Please check out  ExecuTorch.

File: beginner_source/deeplabv3_on_android.rst [hunk 2]
```diff
@@ -1,234 +1,10 @@
 Image Segmentation DeepLabV3 on Android
 =================================================
 
-**Author**: `Jeff Tang <https://github.com/jeffxtang>`_
+PyTorch Mobile is no longer actively supported. Please check out Executorch.
```
Comment (svekars) [inline beginner_source/deeplabv3_on_android.rst:4]:
```suggestion
PyTorch Mobile is no longer actively supported. Please check out ExecuTorch.
```

File: beginner_source/deeplabv3_on_android.rst [hunk 3]
```diff
@@ -1,234 +1,10 @@
 Image Segmentation DeepLabV3 on Android
 =================================================
 
-**Author**: `Jeff Tang <https://github.com/jeffxtang>`_
+PyTorch Mobile is no longer actively supported. Please check out ExecuTorch.
```
Comment (svekars) [inline beginner_source/deeplabv3_on_android.rst:4]: The spellcheck still doesn't like it, let's make sure we spell ExecuTorch everywhere and add it to the https://github.com/pytorch/tutorials/blob/main/en-wordlist.txt

**Review by byjlw — APPROVED** [review]

**Review by svekars — APPROVED** [review]
Looks good! Can you make sure that all pages are redirecting correctly in the preview. For example: https://docs-preview.pytorch.org/pytorch/tutorials/3121/beginner/deeplabv3_on_android.html - After that good to merge.

**Comment by AlannaBurke** [comment]
@svekars Checked each page and the redirect is working on each one.

**Pull request merged**